packages/design-system/src/components/Pagination/__snapshots__/Page.test.tsx.snap

```diff
@@ -1,25 +1,28 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Page should render interactive el if not current 1`] = `
-<li>
-  <Button
-    aria-label="page 1"
-    href="/#1"
-    onClick={[MockFunction]}
-    variation="ghost"
-  >
-    1
-  </Button>
-</li>
+<div>
```

Comment:
Something I noticed doing my tests was RTL adds an extra wrapping div to components. I don't know if this is an issue or not with our snapshots, but it's worth calling out that this wrapping div may not be part of the actual component we're testing.

Reply:
Good call. I updated to target the element that `Page` actually renders. This makes it more consistent with your accordion tests.

Reply:
@zarahzachz, if you use `container` to take the snapshot, that container defaults to a `div` and will therefore wrap your snapshot in a div. If you use `asFragment()`, which we do in some of our other tests, the wrapping element in the snapshots will be a `<DocumentFragment>`. I'm a fan of trying to target the specific element you want to snapshot in general, as Anna suggested; it makes for smaller snapshots with less noise.